### PR TITLE
Bump rococo relay and coretime-rococo to 1.6

### PR DESCRIPTION
**Pull request by seadanda**

### CODE REVIEW

**Comment by seadanda** [comment]
bot bench-all cumulus --runtime=coretime-rococo

File: cumulus/parachains/runtimes/coretime/coretime-rococo/src/lib.rs [hunk 1]
```diff
@@ -129,7 +129,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("coretime-rococo"),
 	impl_name: create_runtime_str!("coretime-rococo"),
 	authoring_version: 1,
-	spec_version: 1_005_004,
+	spec_version: 1_006_001,
```
Comment (ggwpez) [inline cumulus/parachains/runtimes/coretime/coretime-rococo/src/lib.rs:132]: Why is it 1.6.1?

Comment (seadanda) [reply cumulus/parachains/runtimes/coretime/coretime-rococo/src/lib.rs:132]: when we did the relay chain upgrade, the migration failed and so we had to bump

Comment (ggwpez) [reply cumulus/parachains/runtimes/coretime/coretime-rococo/src/lib.rs:132]: Did it work the second time or will we have to bump again?

**Review by ggwpez — APPROVED** [review]

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/4901419

File: cumulus/parachains/runtimes/coretime/coretime-rococo/src/lib.rs (hunk 1 above)
Comment (seadanda) [reply cumulus/parachains/runtimes/coretime/coretime-rococo/src/lib.rs:132]: It worked, we have it running and configured on rococo and the sales have started (with 0 cores for this sale period).

**Comment by seadanda** [comment]
bot clean
reverting bench

**Comment by bkchr** [comment]
@seadanda are you fixing this?

**Pull request merged**